lib/stat_tracker.rb

```diff
@@ -179,6 +231,14 @@ def win_percentage_per_team
     games_won_per_team.merge(total_games_per_team){|team_id, wins, games| (wins.to_f / games).round(2)}
   end
 
+  def win_percentage_by_season(team_id)
+    result = {}
+    @game_teams.each do |game_team|
+      result[game_team.season] = win_percentage_per_team
+    end
+    result
+  end
```

Comment:
I found a few odd things with this method:
- It takes a team_id argument but doesn't use it
- It gives the same percentages for each season
- There is no test for it that I could find
- It could be refactored with reduce

worst_coach used this method, was it passing the spec harness? It's now taking forever for it to run.